src/commands/getCommand.ts

```diff
@@ -0,0 +1,27 @@
+import Base from './base'
+import { Flags } from '@oclif/core'
+
+export default abstract class GetCommand extends Base {
+    authRequired = true
+
+    static flags = {
+        ...Base.flags,
+        'sortBy': Flags.string({
```

Comment:
should we just sort by name by default and not need to specify in the command? since this will likely need a breaking change if it changes the behaviour of the default gets if calling `get` will prompt the user